website/thaliawebsite/settings.py

```diff
@@ -514,9 +514,13 @@ def from_env(
 CORS_URLS_REGEX = r"^/(?:api/v1|api/v2|user/oauth)/.*"
 
 # OAuth configuration
+OIDC_RSA_PRIVATE_KEY = from_env("OIDC_RSA_PRIVATE_KEY", testing=None)
+if OIDC_RSA_PRIVATE_KEY is not None:
+    OIDC_RSA_PRIVATE_KEY = base64.urlsafe_b64decode(OIDC_RSA_PRIVATE_KEY)
```

Comment:
```suggestion
    OIDC_RSA_PRIVATE_KEY = base64.b64decode(OIDC_RSA_PRIVATE_KEY)
```

This is easier to create as it's used by the `base64` command line program

Reply:
You can change this back because it's better to have it in line with the rest of the base64 decodes

Reply:
Done!